app.py

```diff
@@ -249,27 +279,60 @@ def _get_featured_post(groups=[], categories=[], per_page=1):
 
     return posts[0] if posts else None
 
+def _get_category_by_id(category_id):
+    global CATEGORIESBYID
+    category = CATEGORIESBYID[category_id]['name']
```

Comment:
Shouldn't this return the object containing name and slug for consistence with the other get functions.